File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int num = x;
        int reversed  = 0;
        while(num>0){
            reversed*=10;
            reversed+=(num%10);
            num /= 10;
        }

        return (x==reversed) ? true : false;
        
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        num=x
        reverse = 0
        while num > 0 :
            reverse *= 10
            reverse += (num%10)
            num /= 10
    
        if reverse==x :
            return True
        else : return False
        